### PR TITLE
Reduces spam of /op

### PR DESCRIPTION
**Pull request by LegendIsAwesomes**
Title says it all.

### CODE REVIEW

**Comment by Wild1145** [comment]
Resolves which issue?

**Comment by LegendIsAwesomes** [comment]
@Wild1145 
https://github.com/TotalFreedom/TotalFreedomMod/issues/538

**Comment by Wild1145** [comment]
@LegendIsAwesomes Ty, in future try to include "Resolves #< ISSUE NUMBER >" in the commit description :)

**Comment by LegendIsAwesomes** [comment]
@Wild1145 Okay, Hopefully this is implemented.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java [hunk 1]
```diff
@@ -50,9 +50,18 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
...
-        TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
-        player.setOp(true);
+        
+        
+        if (!player.isOp())
++        {
++        	TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
++        	player.setOp(true);
++        }
+        //Checks if the player is already op.
```
Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java]: Would probably remove this line if I am being picky, while I do like comments, its kinda out of place here as there are none others around it to support it.

**Comment by Wild1145** [comment]
@JeromSar Looks good to me bar one potential issue that I flagged as a line note.

**Comment by LegendIsAwesomes** [comment]
@Wild1145 Sorry i kinda abandoned this, forgot about it. That removes the comment.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java [hunk 2]
```diff
@@ -50,9 +50,17 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
...
+        
+        
+        if (!player.isOp())
++        {
++        	TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
++        	player.setOp(true);
++        }
++        else
++        {
++        	playerMsg("The player you are trying to OP is already OP!");
```
Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java]: Might I suggest changing this to something like 
playerMsg(player.getName() + " is already OP!");

(I havent put that into an IDE so you will need to, but that just looks cleaner I think)

**Comment by Wild1145** [comment]
@LegendIsAwesomes No worries, I just made one final comment on there, once thats addressed (Or if @JeromSar feels its unnecessary) it looks good to go.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java [hunk 3]
```diff
@@ -50,9 +50,17 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
                 return true;
             }
         }
-
-        TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
-        player.setOp(true);
+        
+        
+        if (!player.isOp())
++        {
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java]: Random plus signs? Do you even use a IDE to compile your code. What about testing?

Will you _please_ read the contributing guidelines!

Comment (AlexFreedomMod) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java]: It should be , not + I think

Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java]: It looks like its been coppied from another commit, because Github adds them in...

**Comment by LegendIsAwesomes** [comment]
Changed and removed the +'s.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java [hunk 4]
```diff
@@ -50,9 +50,17 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
...
+        
+        
+        if (!player.isOp())
+        {
+        	TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
+        	player.setOp(true);
+        }
+        else
+        {
+        	playerMsg("The player you are trying to OP is already OP!");
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java]: Please change this to "That player is already OP."

**Comment by JeromSar** [comment]
@LegendIsAwesomes Just a small tweak, and then we can merge this. :)

**Comment by LegendIsAwesomes** [comment]
@JeromSar I deleted my fork of this so I cannot edit without making another PR.

**Comment by JeromSar** [comment]
@LegendIsAwesomes That was not smart thing to do.

**Comment by LegendIsAwesomes** [comment]
@JeromSar  Shall i create a new PR?

**Pull request closed without merging**